cmd/api/src/database/migration/migrations/v7.4.0.sql

```diff
@@ -34,4 +34,43 @@ CREATE TABLE IF NOT EXISTS asset_group_tag_selector_nodes
 	updated_at timestamp with time zone,
 	CONSTRAINT fk_asset_group_tag_selectors_asset_group_tag_selector_nodes FOREIGN KEY (selector_id) REFERENCES asset_group_tag_selectors(id) ON DELETE CASCADE,
 	PRIMARY KEY (selector_id, node_id)
-	);
+);
+
+-- Migrate existing Tier Zero selectors
+WITH inserted_selector AS (
+INSERT INTO asset_group_tag_selectors (asset_group_tag_id, created_at, created_by, updated_at, updated_by, name, description, is_default, allow_disable, auto_certify)
+SELECT agt.id, current_timestamp, 'SYSTEM', current_timestamp, 'SYSTEM', s.name, s.selector, false, true, false
```

Comment:
I'm not sure which way is more readable, but `agt.id` could be a subquery so it's not part of the join and makes it more clear that it's static
```suggestion
SELECT (SELECT name FROM asset_group_tags WHERE name = 'Tier Zero'), current_timestamp, 'SYSTEM', current_timestamp, 'SYSTEM', s.name, s.selector, false, true, false
```